src/pages/ProfilesExplorerView/components/SceneExploreServiceFlameGraph/components/SceneFunctionDetailsPanel/components/GitHubContextProvider/infrastructure/GitSessionCookieManager.ts

```diff
@@ -25,11 +27,15 @@ class InternalGitSessionCookieManager implements GitSessionCookieManager {
   }
 
   setCookie(cookie: string): void {
-    if (!cookie.startsWith(`${GITHUB_SESSION_COOKIE_NAME}=`)) {
+    if (
+      !cookie.startsWith(`${GITHUB_SESSION_COOKIE_NAME}=`) &&
+      !cookie.startsWith(`${NEW_GITHUB_SESSION_COOKIE_NAME}=`)
+    ) {
+      // If incoming cookie is not named, we use the old cookie name
```

Comment:
Is this what we want? It seems to make sense to use the new cookie name if we have an unnamed cookie value.

Reply:
I'm happy to change this to use the new name.

But note that any name we choose here is kind of wrong:
If a unnamed cookie is found, we don't know whether we have a new (pyroscope_git_session) or old (GitSession) backend.

Said that, I don't really know when we can have a unnamed cookie. Maybe I miss something.